xbmc/video/tags/SetTagLoaderNFO.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "SetTagLoaderNFO.h"
+
+#include "FileItem.h"
+#include "NfoFile.h"
+#include "URL.h"
+#include "utils/FileUtils.h"
+#include "utils/URIUtils.h"
+#include "utils/log.h"
+#include "video/SetInfoTag.h"
+#include "video/VideoInfoScanner.h"
+#include "video/VideoInfoTag.h"
+
+#include <utility>
+
+using namespace XFILE;
+
+CSetTagLoaderNFO::CSetTagLoaderNFO(const CFileItem& item, ADDON::ScraperPtr info, bool lookInFolder)
+  : ISetInfoTagLoader(item, std::move(info))
+{
+  if (!m_item.GetVideoInfoTag()->m_set.title.empty() &&
+      !VIDEO::CVideoInfoScanner::GetMovieSetInfoFolder(m_item.GetVideoInfoTag()->m_set.title)
+           .empty())
+    m_path = URIUtils::AddFileToFolder(
+        VIDEO::CVideoInfoScanner::GetMovieSetInfoFolder(m_item.GetVideoInfoTag()->m_set.title),
+        "set.nfo");
+}
+
+bool CSetTagLoaderNFO::HasInfo() const
+{
+  return !m_path.empty() && CFileUtils::Exists(m_path);
+}
+
+CInfoScanner::INFO_TYPE CSetTagLoaderNFO::Load(CSetInfoTag& tag,
+                                               const bool prioritise,
+                                               std::vector<EmbeddedArt>*)
+{
+  CNfoFile nfoReader;
+  CInfoScanner::INFO_TYPE result = CInfoScanner::NO_NFO;
+  result = nfoReader.Create(m_path, m_info);
+
+  if (result == CInfoScanner::FULL_NFO || result == CInfoScanner::COMBINED_NFO)
+    nfoReader.GetDetails(tag, nullptr);
+
+  // ** Get set name from originally passed
+  if (tag.GetTitle().empty())
+    tag.SetTitle(m_item.GetVideoInfoTag()->m_set.title);
+
+  std::string type;
+  switch (result)
+  {
+    case CInfoScanner::COMBINED_NFO:
+      type = "mixed";
+      break;
+    case CInfoScanner::FULL_NFO:
+      type = "full";
+      break;
+    case CInfoScanner::URL_NFO:
+      type = "URL";
+      break;
+    case CInfoScanner::NO_NFO:
+      type = "";
+      break;
+    case CInfoScanner::OVERRIDE_NFO:
+      type = "override";
+      break;
+    default:
+      type = "malformed";
+  }
+  if (result != CInfoScanner::NO_NFO)
+    CLog::Log(LOGDEBUG, "VideoInfoScanner: Found matching {} NFO file: {}", type,
```

Comment:
`Clog::LogF`to log the function name, remove "VideoInfoScanner: ". Here and everywhere else you log something you should consider to switch to `LogF`